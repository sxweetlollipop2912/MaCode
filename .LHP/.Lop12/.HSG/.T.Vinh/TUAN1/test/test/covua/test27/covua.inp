30
x00d11d1d1dd.101d1.d101dd1dd01
1x1d010110d11101d0.110d0d1d101
10x01d1ddd.d.d1.10dd11d11dd..0
dd1x11101.1dd0d0001011d1d01ddd
0100xd11dd0.00dd.00d0d0110d0d0
00d0dxdd1d1.0dd11..d0dddd11010
d1000dx10dd010.000dd1dd1d.1dd1
00d10d0x11010d1d.10100d0d00d11
d0d0d010x.0.0.0001d0d011ddd11d
01d.ddd0.x..1000d.0d1d00d110dd
dd.010d11.xdd1d00d01dd1d1ddd.0
d0dd..10..dxddd1..10d00ddddd0d
.0.d110110ddxd0d00d1011000d.10
00d11d1d.10ddx1d001ddd101000d0
110ddd.011dd10x00ddd10d11d1d00
00.1d01d1110dd1x101d0001d111d1
dd01.01.1d1.1110xd101dd00d1010
01111.100.d.11d1dxdd.0111.10d0
..d01.d1d110d0d00dx10000010011
d0d1ddd01d010ddd1d0x01...0.0d.
00001101d0dd1d010.11x10001.dd0
1100ddd11dd10d11d1100x111111.0
0ddd1ddd010100d1d01.10xddd10d.
d1000d0101dd1100101.10dx01d101
dd0d0ddddd0d100d101.10d1x10110
00d110.1d0dd11d0d.0100d00xd0d1
ddd0d001d0ddd100001..00d1dx101
d0.d11dd01dd.1d01111d010010xd.
11.dd0d00d.10d1d0d0dd.d10d1dx0
001d1100dd1d1110110.11.0100.1x