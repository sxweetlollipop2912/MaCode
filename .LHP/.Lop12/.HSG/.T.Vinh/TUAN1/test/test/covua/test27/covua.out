1 3 18 22
